lib/cloudregister/registerutils.py

```diff
@@ -105,9 +111,10 @@ def clean_hosts_file(domain_name):
         if b'# Added by SMT' in entry:
             smt_announce_found = True
             continue
-        if smt_announce_found and domain_name in entry:
-            smt_announce_found = False
-            continue
+        if smt_announce_found:
+            if domain_name in entry:
```

Comment:
we could check whether `b'smt'` and `b'registry'` are inside `entry` in case there are more entries with the same domain added by the user, as we also ignore the [entries with the same domain before the comment added by SMT](https://github.com/SUSE-Enceladus/cloud-regionsrv-client/blob/cc2603406183c585bc6ae78d46f22a70a80af122/tests/test_registerutils.py#L325)

Reply:
will resolve this conversation unless any objection